app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -87,36 +89,49 @@ object DraftController {
         }?.localUuid
     }
 
-    fun MutableRealm.setDraftSignature(draftLocalUuid: String) {
-        updateDraft(draftLocalUuid, this) { draft ->
+    fun MutableRealm.setSignature(draft: Draft) {
 
-            draft.mimeType = ClipDescription.MIMETYPE_TEXT_HTML
+        draft.mimeType = ClipDescription.MIMETYPE_TEXT_HTML
```

Comment:
Why not init it directly in the constructor?

Reply:
Done.